jupyterhub/notebook-images/overlays/cuda/tensorflow-gpu-notebook.yaml

```diff
@@ -23,6 +23,10 @@ items:
         name: s2i-minimal-notebook-gpu:3.6
     resources:
       limits:
```

Comment:
I think we need to parameterize these "default" resource values so that it can be modified by the user in the kfdef. I think we originally had these hardcoded in the old operator but user's were reporting issues with failed builds when we set the limits which is why we removed the `limits`

Reply:
This image is fine - it is the same as building the s2i-minimal-notebook image, so even these limits are probably unnecessarily high.

For the image below (s2i-tensorflow-gpu-notebook), I'd vote for leaving the limits there as they are set now - min 2 Gi, max 4 Gi - it "should" be enough. We can parameterize later if we see people reporting issues - WDYT @LaVLaS

Reply:
I understand the point, I will remove the limit from this one and leave the s2i-tensorflow-gpu-notebook one. 
 I had added them based on the build I tested in the cluster.

Reply:
We had an offline discussion and based on your experience with gpu notebook builds, there is no current need to add parameters for these values. We can revisit this in the event that a user has a need to modify these values.